SR2021/2020-11-13-sr2021-kickstart-details.md

```diff
@@ -0,0 +1,28 @@
+---
+to: Potential team leaders + signed up teams
+subject: Student Robotics Kickstart Information
+---
+
+Hello there!
+
+We're pleased to announce some more details for the Student Robotics 2020 kickstart.
+
+As you know, following on from the success of our [previous virtual competition](https://studentrobotics.org/news/2020-07-25-post-competition/), we will be running the competition in a simulated environment this year.
+
+## Kickstart
+
+Kickstart will be streamed live on **21st November 2020 at 13:00 (UK Time)**, aiming to finish around 16:00. Here we'll announce the game for this year, go into further details on how the competition year will work, and of course answer any questions you have.
+
+If you've signed up for the competition, you'll receive the access details for your team. With these you can create user accounts for your competitors to give them access to the IDE and forum. We recommend creating accounts for competitors before kickstart to ensure they can get started straight away.
+
+The simulator will be based in [webots](https://cyberbotics.com/#download). The download is large, and we recommend downloading it before the event to ensure you can get started immediately. Information on downloading our simulation will be made available on [the docs](https://studentrobotics.org/docs/competition-simulator/) during the event.
+
+During the event, we recommend setting up a call for you and your team to communicate and collaborate, both to share ideas and discuss potential questions.
```

Comment:
I think we need more detail here as well as a clearer message that we will only be able to help the team during the microgames if a team-leader is present in the call.
I'm assuming that we've had the plan of having the team-leader be the one to invite us into their chats OK'd by @Scarzy?

Reply:
It hadn't. I can't think of anything wrong with that approach at this time, it does at least ensure the team leader is in the call at the time.

Incidentally, I imagine most team leaders will be setting these up as Microsoft Teams calls, so we'd need volunteers to be able to support that.

Reply:
We've changed this to do the comms over discord instead, as it's more public from a volunteer perspective, and there's less organisation needed around accounts / scheduling (as you say we'd perhaps need volunteers to have O365 accounts or something!)